### PR TITLE
New package: zstd v1.3.8

### PR DESCRIPTION
**Pull request by OTLabs**
New package: zstd v1.3.7

Zstandard - Fast real-time compression algorithm
https://facebook.github.io/zstd/

### CODE REVIEW

**Comment by Alexpux** [comment]
@OTLabs why we need it in MSYS-packages? We already have it under mingw

**Comment by OTLabs** [comment]
@Alexpux It's a very good question.

I see that there are packages for xz under mingw and msys same time, tar and gzip are msys packages only. I thought that it would be appropriate to have this compression utility in msys too.

I was thinking that mingw and msys are two separate systems so we need to have packages in each of them.

I would quite appreciate your comments about how to maintain same package in msys and/or mingw.

File: zstd/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX="/usr" \
+    ../${pkgname}-${pkgver}/build/cmake
+
+  make install DESTDIR="${srcdir}/dest"
+}
+
+package_zstd() {
+  groups=('libraries')
+  depends=('gcc-libs' "zstd=${pkgver}")
```
Comment (Alexpux) [inline zstd/PKGBUILD]: packages cant depends on itself. I think you mean libzstd

Comment (OTLabs) [reply zstd/PKGBUILD]: Done! Thank you for the tip.

**Pull request merged**